selfdrive/car/hyundai/hyundaican.py

```diff
@@ -38,7 +38,7 @@ def create_lkas11(packer, frame, car_fingerprint, apply_steer, steer_req,
                          CAR.ELANTRA_HEV_2021, CAR.SONATA_HYBRID, CAR.KONA_EV, CAR.KONA_HEV, CAR.KONA_EV_2022,
                          CAR.SANTA_FE_2022, CAR.KIA_K5_2021, CAR.IONIQ_HEV_2022, CAR.SANTA_FE_HEV_2022,
                          CAR.SANTA_FE_PHEV_2022, CAR.KIA_STINGER_2022, CAR.KIA_K5_HEV_2020, CAR.KIA_CEED,
-                         CAR.AZERA_6TH_GEN):
+                         CAR.AZERA_6TH_GEN, CAR.CUSTIN_1ST_GEN):
```

Comment:
What does the dash display without this?

Reply:
@sshane  IIRC, I don't think it displays anything, but EPS would fault and does not allow steering.